docs/cluster/troubleshooting_a_cluster.asciidoc

```diff
@@ -50,42 +63,89 @@ systemctl start packetfence-galera-autofix
 
 This action will not cause service disruption on current cluster.
 
-=== Recovering when a node is missing
-
-If one of the nodes cannot recover, you can manually reset the cluster state. Note that this procedure is only valid if all the nodes were hard-shutdown at the same time.
+After all nodes have joined back cluster, you should verify <<_checking_the_mariadb_sync,MariaDB sync>>.
 
-First, stop `packetfence-mariadb` on all servers:
+WARNING: After stopping `packetfence-mariadb` service, be sure there is no more `mysql` process running.
 
-  # systemctl stop packetfence-mariadb
+=== No more database service
 
-Then remove `/var/lib/mysql/gvwstate.dat` on all the servers that are still alive.
+When there is no more database service in your cluster, you need to do a full recovery.
 
-Next, see the section 'Recovering from a split brain' to start up a new cluster.
+You must identify the node you wish to keep the data from and start it with the
+`--force-new-cluster` option.
 
-=== Recovering from a split brain
+Node which has the most recent data will have a highest `seqno` value in
```

Comment:
```suggestion
Find the node which has the most recent data will have a highest `seqno` value in
```

Reply:
I could be wrong but your suggestion looks weird to me.

Reply:
> I could be wrong but your suggestion looks weird to me.

Yep, just adjusted that